runtime/doc/vi_diff.txt

```diff
@@ -57,10 +57,9 @@ argument when starting Vim.
 
 Support for different systems.
 	Vim can be used on:
-	- All Unix systems (it works on all systems it was tested on, although
-	  the GUI and Perl interface may not work everywhere).
-	- Windows 95 and Windows NT, with support for long file names.
-	- Macintosh
+	- Modern Unix systems (*BSD, Linux, etc.)
+	- Windows
```

Comment:
Sure. I would like to drop XP support (since MS dropped it too). It makes things like symlinks easier because Vista+ has them.

Anyways will update to this for now.